packages/create-react-app-demo/src/docs-chat/ai.tsx

```diff
@@ -1,32 +1,50 @@
 /** @jsx AI.createElement */
 /** @jsxFrag AI.Fragment */
 import * as AI from 'ai-jsx';
-import { DocsQA, ScoredChunk, LocalCorpus, defaultChunker, staticLoader } from 'ai-jsx/batteries/docs';
-// @ts-expect-error
-import markdownPath from './brand-new.md';
+import { DocsQA, DefaultFormatter, LocalCorpus, defaultChunker, staticLoader } from 'ai-jsx/batteries/docs';
 
-// For now, we load the ai.jsx docs from a local markdown file. Once we have a HTML->Markdown converter,
-// we can load the docs from the site directly.
-const docResponse = await fetch(markdownPath);
-const docText = await docResponse.text();
-const docs = [
-  {
-    pageContent: [docText],
-    name: 'Guide for AI Newcomers',
-  },
-];
-const corpus = new LocalCorpus(staticLoader(docs), defaultChunker);
+let globalCorpus: LocalCorpus | undefined;
 
-const corpusLoadedPromise = corpus.startLoading();
-
-const ShowDoc = ({ doc }: { doc: ScoredChunk }) => (
-  <>
-    Title: {doc.chunk.documentName ?? 'Untitled'}
-    Content: {doc.chunk.content}
-  </>
-);
+/**
+ * This is a very simple example of how to build a document corpus.
+ * We pull markdown content from a set of URLs and then index it.
+ */
+export async function indexCorpus() {
+  const files = {
+    'getting-started.md': 'Getting Started',
+    'guides/ai-ui.md': 'AI + UI',
+    'guides/brand-new.md': 'Guide for AI Newcomers',
+    'guides/docsqa.md': 'Docs QA: Grounding Answers',
+    'guides/esm.md': 'ESM',
+    'guides/jsx.md': 'JSX: Build System Considerations',
+    'guides/observability.md': 'Observability',
+    'guides/prompting.md': 'Getting the AI to say things',
+    'guides/rules-of-jsx.md': 'Rules of AI.JSX',
+  };
```

Comment:
I feel much better about this demo now that we're reading all the files. :smile: